pkg/graph/crd/crd.go

```diff
@@ -101,3 +101,11 @@ func newCRDSchema(spec, status extv1.JSONSchemaProps, statusFieldsOverride bool)
 		},
 	}
 }
+
+func newCRDAdditionalPrinterColumns(additionalPrinterColumns []extv1.CustomResourceColumnDefinition) []extv1.CustomResourceColumnDefinition {
+	if len(additionalPrinterColumns) == 0 {
+		return defaultAdditionalPrinterColumns
+	}
+
+	return additionalPrinterColumns
+}
```

Comment:
Thought: what are your thoughts in injecting specific default `CustomResourceColumnDefinition` when they don't exist, that way we ensure that STATE/SYNCED/AGE are always included?

Reply:
My reasoning was that users might not need the default columns and might want to remove them. If they are already customizing printer columns, my guess is that they want to show a specific state of their CRs.

However, I don't have such a strong opinion on this, and I can merge the defined additionalPrinterColumns with the default ones based on the `name.` Let me know if you agree with the point above or if we should add the default columns, and I can add them.

Reply:
I think allowing users to override our default printer columns seems reasonable. If they want to maintain the existing ones, they should be able to.

I mentioned above we should call that out in the docs though